src/Aether/Aether.fs

```diff
@@ -110,39 +110,39 @@ module Compose =
 
 (* Lenses
 
-   Various lenses implemented for common types such as tuples,
-   lists and maps, along with an id lens (which is useful for composing
-   a lens which has not specific "lensing" elements but is implicitly a chain
-   of one or more isomorphisms. Having an id lens enables the root composition. *)
+   Various lenses for common types such as tuples,
+   lists, and maps; along with an id_ lens (which is useful for composing
```

Comment:
If the problem is only stylistic I would opt for conforming to the code. In particular, if you want to state the abstract notion of identity, then use "identity"  and not the abbreviation. If you gonna go jargon, might as well use the code as a glossary. 

Please note, I am rather new to FP and the lensing was a big jump. The terminology is counter-intuitive and the documentation kinda of under-sells the notion that the four get and set Lens functions are merely upgrades to the tuple fst and snd functions ('course I could be wrong) for using the associated functions of a Lens.